wiki/Performance_Troubleshooting/en.md

```diff
@@ -86,3 +86,18 @@ While this may seem like the best option (and indeed can be, if your hardware ca
 - GPUs generally aren’t meant to render 1000+ frames per second, so unexpected behaviour may occur, including lower performance or higher latency than otherwise attainable.
 
 Even so, if heat generation and potentially overworking your hardware is not considered an issue, unlimited may be the option that provides you the most stable frame times.
+
+### Custom
```

Comment:
Is there any logic behind adding this to the article? It seems unnecessary and the ability to switch to this frame limiter was removed from in game for a reason. There are very few situations where changing this could ever help performance.

It's also very easy to do incorrectly, regardless of how detailed the instructions are.

Reply:
It has never been removed, it's never in the UI in the first place AFAIK.

Since fallback, there're 5 options: VSync, 120fps, 240fps, Unlimited (gameplay) and Unlimited.

And in latest (before the "Power Saving/Optimal" around last year), there were 4 options, same as above, except the Unlimited.

Although removed, you can still set to `FrameSync = CompletelyUnlimited` at that time, but well, no point in making the song select loads at max FPS.

But it was always 120/240 regardless of your monitor. And the third option, aka 240fps, can be set to a custom number via `FrameSync = 500fps` and `CustomFrameLimit = 500`. Only then, the game options would be: VSync, 120fps, 500fps, Unlimited (gameplay)

Reply:
So the point to set a custom frame limit can vary:
- I want my capped FPS to be a integer multiply of my monitor refresh rate. (let's say 2x. 4x, 8x, but it already got covered since Power Saving/Optimal update)
> GPUs generally aren’t meant to render 1000+ frames per second
- I have 60hz monitor and I want to set at 960, or 1000fps, but Optimal only provides me 480, and Unlimited burns my PC.
- I have a laptop and want to conserve battery, I don't want to consume much power with Optimal or Unlimited, but Power Saving gives me so much latency (because it's not osu!lazer that is multi-threadded), so I want a decent cap, like 4x refresh rate (instead of 2x and 8x marks).
- ...etc

Reply:
Before the settings were changed to optimal and power saving, I believe the 240fps option would just change to whatever you had the custom FPS limit set to. So you were able to switch back and forth between your custom FPS and the other options.

I still think this is unnecessary to leave in, but if you are dead set on getting it added for whatever reason, you need to make sure the steps to changing it are correct and will actually work 100% of the time. Changing the cfg file without closing the game first will not work.

Reply:
> the 240fps option would just change to whatever you had the custom FPS limit set to as you mentioned. So you were able to switch back and forth between your custom FPS and the other options
It only changes the 240 to another number, not creating a new one though. And you would still need to change that manually in the cfg file.
> Changing the cfg file without closing the game first will not work.
Yes, this is true, I'll edit it.
> Opening the file in notepad before closing the game and then changing it may also not work.
You can open the file in the notepad normally, just don't save the file before closing the game.
We just need the value to be there and saved before opening the game.